Formula/c/cfripper.rb

```diff
@@ -2,9 +2,9 @@ class Cfripper < Formula
   include Language::Python::Virtualenv
 
   desc "Library and CLI tool to analyse CloudFormation templates for security issues"
-  homepage "https://cfripper.readthedocs.io/"
```

Comment:
can we also raise an upstream issue on this as well?

Reply:
https://github.com/Skyscanner/cfripper/issues/253